src/core/db/repository/admin_repository.py

```diff
@@ -13,5 +13,5 @@ def __init__(self, session: AsyncSession) -> None:
 
     async def get_by_email(self, email: str) -> AdminUser | None:
         """Возвращает пользователя (или None) по email."""
-        user = await self._session.execute(select(AdminUser).where(AdminUser.email == email))
-        return user.scalars().first()
+        user = await self._session.scalars(select(AdminUser).where(AdminUser.email == email))
+        return user.first()
```

Comment:
Тут можно проще: 
```suggestion
        await self._session.scalar(select(AdminUser).where(AdminUser.email == email))
```
Это не только вопрос "красоты" кода, но и оптимизации - вместо двух действий, делаем одно.

В случае со `scalar` это будет работать также как:
```python
res = await self._session.execute(...)
return res.first()
```